### PR TITLE
Added rate-limit-table

### PR DESCRIPTION
**Pull request by LakshmiLavanyaKasturi**
## Description

Describe the changes that were made in this pull request. When possible start with the motivations behind the change. Be sure to also include the following information:

1. Motivation for change?
2. What was changed?
3. Link to relevant issues?

## Checklist

- [ ] [Conventional commits were used](https://www.conventionalcommits.org/en/v1.0.0/)
- [ ] New links to files and images were verified
- [ ] For fixes/refactors: all existing references were identified and replaced
- [ ] [Style guide](https://developers.google.com/style) was reviewed and applied
- [ ] Clear code samples were provided
- [ ] People were tagged for review

### CODE REVIEW

File: docs/get-started/stacks-blockchain-api.md [hunk 1]
```diff
@@ -157,6 +157,20 @@ await sub.unsubscribe();
 
 Rate limiting is only applied to [faucet requests](https://docs.hiro.so/api#tag/Faucets) and based on the address that tokens are requested for.
 
+You can refer to the rate limit for each endpoint in the table below:
+
+|Endpoint  | Rate-Limit (RPM)|
+| --- | ---|
+|stacks-node-api.stacks.co/extended/|500|
+|stacks-node-api.mainnet.stacks.co/extended/|500|
```
Comment (sabbyanandan) [inline docs/get-started/stacks-blockchain-api.md:165]: Maybe we can sort the endpoint, so all mainnet, testnet and rest all show in an order.

**Review by sabbyanandan — CHANGES_REQUESTED** [review]
@CharlieC3 @deantchi: Do you have any other thoughts to share?

File: docs/get-started/stacks-blockchain-api.md (hunk 1 above)
Comment (LakshmiLavanyaKasturi) [reply docs/get-started/stacks-blockchain-api.md:164]: Done. Endpoints are sorted now.

**Comment by LakshmiLavanyaKasturi** [comment]
@CharlieC3 and @sabbyanandan: Please review my changes and let me know if this PR is good to merge. Thanks.

**Comment by CharlieC3** [comment]
@LakshmiLavanyaKasturi Looks like the `mainnet` and no-name URLs are still separated:
https://hiro-systems-docs-ih87bdjs8-blockstack.vercel.app/get-started/stacks-blockchain-api#rate-limiting

I could clarify a bit more. Since URLs like `stacks-node-api.mainnet.stacks.co/extended/` and `stacks-node-api.stacks.co/extended/` point to the same backend, they will always have the same rate-limit configuration in place. So instead of placing them in separate rows, can we place each pairing in one row instead?

**Comment by LakshmiLavanyaKasturi** [comment]
@CharlieC3 Can you please review now. Thanks!

**Comment by CharlieC3** [comment]
@LakshmiLavanyaKasturi Looks like all of the mainnet + no-names are combined in one row, and the testnet URLs combined in another. I'm looking only to pair up the mainnet + no name URLs on similar _paths_. The testnet URLs should have their separate row like before. e.g.
```
stacks-node-api.mainnet.stacks.co/extended/
or
stacks-node-api.stacks.co/extended/
```

```
stacks-node-api.mainnet.stacks.co/rosetta/
or
stacks-node-api.stacks.co/rosetta/
```

etc

**Review by CharlieC3 — APPROVED** [review]
LGTM! :shipit:  Thanks for bearing with me on the changes! 🙏

**Pull request merged**